articles/quickstart/native/android/interactive.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Add Login to your Android App
+description: This quickstart demonstrates how to add user login to an Android application using Auth0.
+seo_alias: android
+interactive: true
+files:
+  - files/build
+  - files/strings
+  - files/main
+github:
+  path: 00-Login-Kt
+---
+
+# Add Login to your Android App
+
+<!-- markdownlint-disable MD002 MD041 -->
+
+<!-- markdownlint-disable MD002 MD034 MD041 -->
+
+<%= include('../_includes/_getting_started', { library: 'Android', callback: 'http://localhost:4200', returnTo: 'http://localhost:4200', webOriginUrl: 'http://localhost:4200', showLogoutInfo: true, showWebOriginInfo: false, new_js_sdk: false, show_install_info: false }) %>
+
+<%= include('../../../_includes/_callback_url') %>
```

Comment:
Would it make sense to use the `../../_includes/_configure_auth0_interactive.md` here? That one refers to the interactive application configurator that is part of the new design. Users can now setup apps and its settings through that.

Reply:
@frederikprijck we use `account.namespace` for callback URLs and this doesn't seem to be supported in `_configure_auth0_interactive` so I have used the existing one. I have changed the description for configuring application with same description from the `_configure_auth0_interactive`